src/Apitizer/Types/Filter.php

```diff
@@ -90,7 +90,7 @@ public function handleUsing(callable $handler): self
     /**
      * Filter by field and operator.
      *
-     * If `expectMany` is used, the operator will be ignored in favour of a
+     * If `expectArray` is used, the operator will be ignored in favour of a
```

Comment:
`expectArray` is an implementation detail that probably shouldn't be used in the documentation. Since `expectMany` also no longer exists, prefer something like `If an array of inputs is expected...`